chat/message_history.py

```diff
@@ -0,0 +1,30 @@
+import uuid
+
+from skygear.container import SkygearContainer
+from skygear.options import options as skyoptions
+from skygear.transmitter.encoding import serialize_record
+from skygear.utils.context import current_user_id
+
+
+class MessageHistory:
+    def __init__(self, message):
+        message_record = serialize_record(message.record)
+        self.record = {}
+        for key in ['attachment', 'body', 'metadata',
```

Comment:
Cannot, because this will also copy other unwanted values such as seq and deleted field